src/relation.js

```diff
@@ -450,8 +433,14 @@ class Relation {
      * @public
      */
     dispose () {
-        this._parent.removeChild(this);
+        this._parent && this._parent.removeChild(this);
         this._parent = null;
+        this._children.forEach((child) => {
+            if (child._parent === this) {
```

Comment:
Why is this check is needed?

Reply:
I implemented based on old version of addParent, now it is updated.